cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -373,6 +373,26 @@ def create_service_configs(kube_control):
     remove_state('authentication.setup')
 
 
+@when('kube-control.departed')
+@when('leadership.is_leader')
+def flush_auth_for_departed(kube_control):
+    ''' Unit has left the cluster and needs to have its authentication
+    tokens removed from the token registry '''
+    token_auth_file = '/root/cdk/known_tokens.csv'
+    departing_unit = kube_control.flush_departed()
+    known_tokens = open(token_auth_file, 'r').readlines()
+    for line in known_tokens:
+        if departing_unit in line:
```

Comment:
Is this safe? Would unit `kubernetes-worker/1` also match `kubernetes-worker/11`?